### ISSUE TITLE
Pattern fill only shows up once

### ISSUE THREAD

**Issue by ggl** [highcharts/highcharts#8658]
If one uses several graphs with pattern fills in different Bootstrap tabs (show/hide divs), the pattern only show up in the visible tab. 

#### Expected behaviour

Patterns should show up in all tabs.

#### Actual behaviour

The pattern shows up only once in the displayed tab. Once the graph where the pattern shows up is destroyed, the pattern shows up in the other tab. Same behaviour on predefined patterns like:

    color: 'url(#highcharts-default-pattern-1)',

#### Live demo with steps to reproduce

https://jsfiddle.net/L00ytrcb/339/
Change tab.

#### Product version

Highcharts 6.1.1

#### Affected browser(s)

Chrome 67.0.3396.99 and Firefox 61.0.1, probably others as well.

**Comment by sebastianbochan** [comment]
Hi @ggl, @mdomnita,
Thank you for the reporting.

**Comment by pawelfus** [comment]
**Workaround:**
Define ID for each pattern: https://jsfiddle.net/BlackLabel/L00ytrcb/355/

*Internal note:*
Both charts use the same patterns. It looks like ID's generator (`hashFromObject`) does not prevent this collision.

**Comment by mdomnita** [comment]
Thanks for the workaround, @pawelfus .
This is fine if the pattern is defined by the user but for the predefined patterns the id is fixed.

I am talking about those defined like 
   `chart.renderer.addPattern({
    id: 'highcharts-default-pattern-' + i,`
in highcharts/js/modules/pattern-fill.src.js

I used another workaround in my code similar to this:
https://jsfiddle.net/mdomnita/L00ytrcb/379/
Basically, it empties all charts in the page (including the hidden ones) using jQuery and only fills the chart in the visible containers (tab). It's still overkill but it works for the moment.
